### PR TITLE
add alert messages to all pages

### PR DESCRIPTION
**Pull request by nooralnabi**
add successfully and updated alert messages to all pages are available for now

### CODE REVIEW

File: app/views/clients/show.html.haml [hunk 1]
```diff
@@ -1,4 +1,7 @@
-%p#notice= notice
+.container
```
Comment (ssharba) [inline app/views/clients/show.html.haml]: @nooralnabi flash must be in the application layout and shared among the entire application

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -22,7 +22,12 @@
             %li= link_to "#{current_user.name} Sign out?", signout_path(user_id: current_user.id), id: "sign_out"
             %li{class: "#{'active' if @clients}"}= link_to 'Clients', clients_path
             %li{class: "#{'active' if @contractors}"}= link_to 'Contractors', contractors_path
-
+  .container
```
Comment (ssharba) [inline app/views/layouts/application.html.haml]: @nooralnabi small change, just add text-center to your message.

File: app/controllers/contractors_controller.rb [hunk 3]
```diff
@@ -42,7 +42,7 @@ def create
   def update
     respond_to do |format|
       if @contractor.update(contractor_params)
-        format.html { redirect_to @contractor, notice: 'Contractor was successfully updated.' }
+        format.html { redirect_to @contractor, info: 'Contractor was successfully updated.' }
         format.json { render :show, status: :ok, location: @contractor }
       else
         format.html { render :edit }
```
Comment (ssharba) [inline app/controllers/contractors_controller.rb]: else will have error object with error message, try to display it.

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
-
+  add_flash_types :success, :warning, :danger, :info
```
Comment (ssharba) [inline app/controllers/application_controller.rb]: do we need this? see if you can remove it.

File: app/views/clients/_form.html.haml [hunk 5]
```diff
@@ -1,14 +1,16 @@
...
-.col-md-4     
+.col-md-4
+.col-md-4
   .contrac
     %h1 New client
     = form_for @client do |f|
       - if @client.errors.any?
         #error_explanation
-          %h2= "#{pluralize(@client.errors.count, "error")} prohibited this client from being saved:"
+          .alert.alert-danger
```
Comment (ssharba) [inline app/views/clients/_form.html.haml]: @nooralnabi since we have the code in the layouts, no need to have flash message in every page.

Comment (nooralnabi) [reply app/views/clients/_form.html.haml]: @ssharba  this error message is different  this show just when clients create new one or edit

Comment (ssharba) [reply app/views/clients/_form.html.haml]: no need for it here, move the logic to the controller and the layouts will pick it up. no need to show the count just error

Comment (nooralnabi) [reply app/views/clients/_form.html.haml]: @ssharba  this message  created by scaffold I just and style  i will remove it

**Pull request merged**